codebuild.tf

```
@@ -1,12 +1,13 @@
 module "codebuild" {
-  source = "github.com/SPHTech-Platform/terraform-aws-codebuild"
+  source  = "SPHTech-Platform/codebuild/aws"
+  version = "1.1.1"
```

```suggestion
  version = "~> 1.1.1"
```
And regenerate README again. after above change.